config/crd/bases/toolchain.dev.openshift.com_tiertemplates.yaml

```diff
@@ -141,6 +141,7 @@ spec:
                 required:
                 - objects
                 type: object
+                x-kubernetes-embedded-resource: true
```

Comment:
we could, indeed. To be honest, I have not checked if there are other places where this could be needed. This annotation is used to retain the metadata of the embedded resources, that's all.